lib/bibliothecary/parsers/pypi.rb

```diff
@@ -4,7 +4,12 @@ class Pypi
       include Bibliothecary::Analyser
 
       INSTALL_REGEXP = /install_requires\s*=\s*\[([\s\S]*?)\]/
-      REQUIRE_REGEXP = /([a-zA-Z0-9]+[a-zA-Z0-9\-_\.]+)([><=\w\.,]+)?/
+
+      # Capture Group 1 is package.
+      # Optional Group 2 is [extras].
+      # Capture Group 3 is Version
+      REQUIRE_REGEXP = /([a-zA-Z0-9]+[a-zA-Z0-9\-_\.]+)(?:\[.*?\])*([><=\w\.,]+)?/
```

Comment:
`(?:\[.*?\])*` is new between the match groups, as match group 2 that we ignore.